src/main/java/world/bentobox/bentobox/api/panels/PanelItem.java

```diff
@@ -130,8 +130,13 @@ public boolean isGlow() {
     public void setGlow(boolean glow) {
         this.glow = glow;
         if (meta != null) {
-            meta.addEnchant(Enchantment.ARROW_DAMAGE, 0, glow);
+            if (glow) {
+                meta.addEnchant(Enchantment.ARROW_DAMAGE, 0, glow);
+            } else {
+                meta.removeEnchant(Enchantment.ARROW_DAMAGE);
+            }
             icon.setItemMeta(meta);
```

Comment:
Fix for persistent glowing. It seems this is a change to the underlying server code in that a value of false would not cause a glow, but now it does.